sup3r/models/multi_step.py

```diff
@@ -238,7 +253,11 @@ def generate(self, low_res, norm_in=True, un_norm_out=True):
             low_res = self._normalize_input(low_res)
```

Comment:
I think the exogenous data for model 0 needs to be attached to the low res data before normalization

Reply:
good catch!